.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+on: 
+  pull_request:
+  push:
+    
+jobs:
+  jpo-security-svcs:
+   runs-on: ubuntu-latest
+   container:
+     image: openjdk:17-jdk-slim-buster
+     options: --user root
+   steps:
+     - name: Checkout ${{ github.event.repository.name }}
+       uses: actions/checkout@v3    
+     - name: Set up Maven
+       uses: stCarolas/setup-maven@v4.5
+       with:
+        maven-version: 3.8.2          
+     - uses: actions/setup-java@v3
+       with:
+         java-version: '11'
+         distribution: 'temurin'               
+     - name: Build
+       run: |
+         cd $GITHUB_WORKSPACE
+         ls -ls && pwd
+         mvn -e -X clean org.jacoco:jacoco-maven-plugin:prepare-agent package
+         mvn -e -X clean org.jacoco:jacoco-maven-plugin:report package
+     - name: Archive code coverage results
+       uses: actions/upload-artifact@v3
+       with:
+         name: jpo-security-svcs
+         path: /__w/jpo-security-svcs/jpo-security-svcs/target 
+         if-no-files-found: error
+     - name: build tmca-tester
+       uses: actions/setup-node@v3
+       with:
+         node-version: '18.x'
+     - run: |
+         cd /__w/jpo-security-svcs/jpo-security-svcs/tmca-tester
+         npm install node-menu@1.3.2
```

Comment:
Remove this line about node-menu, no need to manually install a dependency already declared in package.json